Solution1/Solution1.Tests/BL/Services/WeatherServiceTests.cs

```diff
@@ -14,22 +21,27 @@ public class WeatherServiceTests
     {
         private readonly IMapper _mapper;
         private readonly WeatherService _weatherService;
+        private readonly IValidator<ForecastWeatherRequestDTO> _validator;
```

Comment:
why it's not a mock?? are you testing validator?

Reply:
+